README.md

```diff
@@ -28,7 +28,7 @@ random actions. This can be used as a base for creating your own agents, and as
 a simple tool to preview an environment.
 
 ```sh
-bazel run -c opt dmlab2d/python:random_agent -- --level_name=clean_up
+bazel run -c opt dmlab2d/random_agent --config=lua5_2  -- --level_name=clean_up
```

Comment:
I would prefer not to prejudice the choice of Lua version here.

If you'd really like to make a note somewhere else, then maybe that could be a separate commit, but note that we don't really support MacOS officially at this point.

Reply:
Thanks for the comment.  Removed the specific version of Lua and kept out references of Mac just to keep it simple.